roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +86,50 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  become: true
```

Comment:
this should not need become as the directory is owned by the ansible user.

Reply:
thanks, done